lib/modules/powershell/lateral_movement/invoke_portfwd.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import print_function
+from builtins import str
+from builtins import object
+from lib.common import helpers
+
+
+class Module(object):
+
+    def __init__(self, mainMenu, params=[]):
+
+        # Metadata info about the module, not modified during runtime
+        self.info = {
+            # Name for the module that will appear in module menus
+            'Name': 'Invoke-PortFwd',
+
+            # List of one or more authors for the module
+            'Author': ['@decoder-it'],
+
+            # More verbose multi-line description of the module
+            'Description': ('Forward a port with no admin rights required.'),
+
```

Comment:
We recently updated the module template to include links to the MITRE ATT&CK techniques and software. Can you add them here with the technique T1363?

```
            'Software': '',

            'Techniques': ['T1363'],
```

Reply:
Done that.